src/pudl/transform/ferc1.py

```diff
@@ -1217,6 +1217,48 @@ class PlantsHydroFerc1TableTransformer(Ferc1AbstractTableTransformer):
 
     table_id: Ferc1TableId = Ferc1TableId.PLANTS_HYDRO_FERC1
 
+    def transform_main(self, df):
+        """Add bespoke removal of duplicate record after standard transform_main."""
+        return super().transform_main(df).pipe(self.targeted_drop_duplicates)
+
+    def targeted_drop_duplicates(self, df):
+        """Targeted removal of known duplicate record.
+
+        There are two records in 2019 with a ``utility_id_ferc1`` of 200 and a
+        ``plant_name_ferc1`` of "marmet". The records are nearly duplicates of
+        eachother, except one have nulls in the capex columns. Surgically remove the
+        record with the nulls.
+        """
+        if 2019 in df.report_year.unique():
```

Comment:
i made this run only if 2019 is being processed, but I could change the assertion below to only yell if `len(possible_dupes) != 2` and `2019 in df.report_year.unique()`

Reply:
I think only having the assertion in the conditional is better -- we want these kinds of spot fixes to run and work regardless of what data is being processed, and having the whole thing enclosed in a conditional suggests that it will *not* work if 2019 isn't in the data.